app.yml

```
@@ -32,7 +32,7 @@ default_events:
 # - project_card
 # - project_column
 # - public
-# - pull_request
+- pull_request
 # - pull_request_review
 # - pull_request_review_comment
 # - push
```

If this refers to a branch push event, this should probably be enabled as well right?
```suggestion
- push
```